engine/src/cmdse.cpp

```diff
@@ -692,7 +684,19 @@ Parse_stat MCMessage::parse(MCScriptPoint &sp)
 			MCperror->add(PE_SEND_BADTARGET, sp);
 			return PS_ERROR;
 		}
-		return gettime(sp, &in, units);
+        
+        if (sp.skip_token(SP_FACTOR, TT_PREP, PT_AS) == PS_NORMAL)
+        {
+            if (!send ||
+                sp.skip_token(SP_FACTOR, TT_PROPERTY, P_SCRIPT) != PS_NORMAL)
+            {
+                MCperror->add(PE_SEND_BADAS, sp);
+                return PS_ERROR;
+            }
+            script = True;
+        }
+        else
+            return gettime(sp, &(&in), units);
```

Comment:
@livecodeali do we want to support send in time as script? I guess we don't have a use case for it at the moment.